batch.py

```diff
@@ -284,7 +284,10 @@ def crm_pipeline(
     # grab all relevant genes across all chromosomes
     # simpler if gene details are condensed to one file
     gene_dict: dict[str, dict] = {}
-    chromosomes_list = chromosomes.split(' ')
+    if chromosomes == 'all':
```

Comment:
can you add default='all' to the click argument so that it matches the help message